src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
```

Comment:
good job, but you have something to fix: you could change names for `const` by `midTerm` to `MID_TERM`. it's not a mistake but there some reasons to to like that: 

1. you can easily notice constants in code and distinguish them from variables.
2. if you see that it's constant, you will not try accidentally change it.
3. and also too many developers agree with this rule, and write like that.